### PR TITLE
Changes to move code to open-ce-builder

### PR DESCRIPTION
**Pull request by bnemanich**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [x] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [x] Did you write any [tests](https://github.com/open-ce/open-ce/blob/main/tests/) to validate this change?  

## Description

Changes needed to move the Open-CE build tools to the new open-ce-builder repo.

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Comment by bnemanich** [comment]
This won't pass tests until the environment files are moved to the `open-ce` repo.

File: README.md [hunk 1]
```diff
@@ -165,7 +143,3 @@ After performing the build using `open-ce build env`, the `open-ce build image`
 
 For contribution information, please see the [CONTRIBUTING.md](CONTRIBUTING.md) page.
 
-### Slack Community
```
Comment (jayfurmanek) [inline README.md:168]: Where are we thinking is best for this link?

**Comment by jayfurmanek** [comment]
This looks good!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ## Checklist before submitting
 
 - [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
-- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
-- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/main/tests/) to validate this change?  
+- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce-builder/blob/main/doc/)?
+- [ ] Did you write any [tests](https://github.com/open-ce/open-ce-builder/blob/main/tests/) to validate this change?
```
Comment (bethune-bryant) [inline .github/pull_request_template.md:5]: We're removing this file, but we should probably make these changes in https://github.com/open-ce/.github

**Review by bethune-bryant — APPROVED** [review]

**Review by jayfurmanek — APPROVED** [review]

**Pull request merged**